outernet/federation/v1alpha/federation.proto

```diff
@@ -71,15 +71,15 @@ option java_package = "com.outernet.federation.v1alpha";
 //   Requestor of Service termination.
 // - CancelService enables Requestor cancellation of an existing
 //   `service_id`.
-service Federation {
+service FederationService {
```

Comment:
My understanding is that those changes are definitely not backwards compatible.

Should the "version" in the MODULE be increased or do we not consider this API "Released" yet, so that we don't care? I guess it's the latter, but just wanted to confirm here.